distribution/sdk/sample-nsili-server/src/main/java/org/codice/alliance/nsili/mockserver/impl/managers/CatalogMgrImpl.java

```diff
@@ -93,7 +96,7 @@ public SubmitQueryRequest submit_query(Query aQuery, String[] result_attributes,
             poa_.activate_object_with_id("submit_query".getBytes(Charset.forName(ENCODING)),
                     submitQueryRequest);
         } catch (ServantAlreadyActive | ObjectAlreadyActive | WrongPolicy e) {
-            System.out.println("submit_query : Unable to activate submitQueryRequest object.");
+            LOGGER.error("submit_query : Unable to activate submitQueryRequest object.", e);
```

Comment:
ERROR seems to high here.  Maybe WARN at highest.  Probably need a better message too.  "Unable to submit query.  Unable to activate..."